snap/snapcraft.yaml

```diff
@@ -254,9 +254,11 @@ parts:
 
   mysql:
     plugin: cmake
-    source: https://github.com/kyrofa/mysql-server.git
-    source-branch: mysql-5.7.18_support_no_setpriority
-    after: [boost]
+    source: https://github.com/mysql/mysql-server
+    source-type: git
```

Comment:
If you use `https://github.com/mysql/mysql-server.git` you don't have to specify the source-type.